.github/styles/config/vocabularies/Docs/accept.txt

```diff
@@ -93,3 +92,19 @@ preconfigured
 asyncio
 parallelization
 IMDb
+iPhone
+iPhones
+iPad
+iPads
+screenshotting
+Fakestore
+SKUs
+SKU
```

Comment:
`SKUs?` should be the same as separate `SKU` and `SKUs` (the same is valid for the other terms)